device/cig/x86_64-cig_cs5435_54p-r0/Cig-CS5435-54P/port_config.ini

```diff
@@ -0,0 +1,55 @@
+# name          lanes             alias             index          speed
+Ethernet0       8                 Ethernet1/1       0              10000
```

Comment:
Normally, we recommend having 'index' match up with front panel port numbering.

Most devices' front panel ports start from number 1. Is this device port number starting from 0?

Reply:
yes, the front panel ports  of our devices start from 1, I just update the file.